A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/A320_Neo_EICAS.js

```diff
@@ -174,51 +180,80 @@ class A320_Neo_EICAS extends Airliners.BaseEICAS {
         this.ACPowerLastState = isACPowerAvailable;
 
         // modification start here
-        var currentAPUMasterState = SimVar.GetSimVarValue("FUELSYSTEM VALVE SWITCH:8", "Bool");  
-        // automaticaly switch to the APU page when apu master switch is on
-        if (this.lastAPUMasterState != currentAPUMasterState && currentAPUMasterState === 1) {  
-            this.lastAPUMasterState = currentAPUMasterState;  
-            this.changePage("APU");
-
-            //if external power is off when turning on apu, only show the apu page for 10 seconds, then the DOOR page
-            if (externalPower === 0) {  
-                this.externalPowerWhenApuMasterOnTimer = 85;
-            }
+        var currentAPUMasterState = SimVar.GetSimVarValue("FUELSYSTEM VALVE SWITCH:8", "Bool");
 
+
+        //Determine displayed page when no button is selected
+        const prevPage = this.pageNameWhenUnselected;
+
+        const altitude = Simplane.getAltitude();
+        const isGearExtended = SimVar.GetSimVarValue("GEAR HANDLE POSITION", "Bool");
+        const currFlightPhase = SimVar.GetSimVarValue("RADIO HEIGHT", "feet") > 1500; //This simvar should be exactly what we need, but it doesn't work: SimVar.GetSimVarValue("L:AIRLINER_FLIGHT_PHASE", "number");
```

Comment:
What is the issue with the flight phase simvar?

Reply:
Hm. Nothing it seems, maybe I made a mistake while testing it out. Will use this instead, but I only tested it with the current version. Should be functionally identical though